doc/guides/device_mgmt/mcumgr.rst

```diff
@@ -209,6 +209,18 @@ on Zephyr. The ones that are supported are described in the following table:
     :kconfig:option:`CONFIG_THREAD_STACK_INFO` and :kconfig:option:`CONFIG_INIT_STACKS` options
     must be set.
 
+.. _mcumgr_jlink_ob_msd:
+
+J-Link MSD Interaction Note
+***************************
+
+Note that on some boards (mostly Nordic, e.g. `nrf52840dk`) which have a J-Link
```

Comment:
Can we reword this, there `Note` just after one in title? And the "mostly Nordic, e.g. `nrf52840dk`" rubs me the wrong way - not our fault...
```
J-Link OB provided Virtual MSD (virtual Mass storage device also known as file drag-and-drop) enabled by default on some development platforms, for example Nordic `nrf52840dk`, may prevent mcumgr from operating correctly, over the CDC port, due to the use of....
```

Reply:
> Was not meant to place blame, just meant nordic for nordic and nordic-like boards as they're the only ones I've seen with or heard mentioned with this issue that have MSD support, will reword.

I am not saying that have you placed a blame, I am just oversensitive to "mostly".
I'd rather state: yes there is an issue, this is a description of the issue, and it may happen to for example to this board; rather than mostly boards from this vendor.
Thanks.

Reply:
Fixed